drgn/helpers/linux/cpumask.py

```diff
@@ -60,3 +60,50 @@ def for_each_possible_cpu(prog: Program) -> Iterator[int]:
 def for_each_present_cpu(prog: Program) -> Iterator[int]:
     """Iterate over all present CPUs."""
     return _for_each_cpu_mask(prog, "__cpu_present_mask")
+
+
+def cpumask_to_cpulist(cpumask: Object) -> str:
+    """
+    Get list of CPUs, present in a cpumask.
+
+    :param cpumask: ``struct cpumask*``
+    :returns: list of CPUs as string
```

Comment:
Let's add some more details and an example:

```suggestion
    Return a CPU mask as a CPU list string.

    >>> cpumask_to_cpulist(mask)
    0-3,8-11

    :param mask: ``struct cpumask *``
    :return: String in the `CPU list format
    <https://man7.org/linux/man-pages/man7/cpuset.7.html#:~:text=List%20format,-The%20List%20Format>`_.
```

Reply:
Done.